MicrosoftFluentUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MicrosoftFluentUI'
-  s.version          = '0.11.0'
+  s.version          = '0.12.0'
```

Comment:
you probably broke subspec because HUD vnext path has changed

Reply:
Yep. Not sure how it stayed in the vnext folder for so long after being brought to main, though.